docs/components/FCard.md

```diff
@@ -35,6 +35,19 @@ Rubriken ska sättas till rätt nivå.
 - Placera inte långa listor eller mycket omfattande information i kort.
 - Gruppera inte information eller formulärskomponeter med kort.
 
+## Felstatus på kort
+
+Det är möjligt att indikera om ett kort är felaktigt genom att aktivera validering (`v-validation`).
+Det går inte att lägga på enskilda validatorer, utan status hanteras helt manuellt av konsumenten.
+Indikering av status sker direkt.
+
+Konsumenten behöver ange det element som ska fokuseras på kopplat till ett fel (`focus-ref`).
+Vanligtvis är detta en knapp för att redigera ett korts innehåll.
+
+```import
+FCardValidationExample.vue
+```
```

Comment:
Hade gärna sett 2-3 rader statisk kod ovanför och utöver exemplet, något likt:

```diff
-<f-card>
+<f-card v-validation>
```

```ts
/* sätter vilket felmeddelande som ska visas */
ValidationService.setError(element, "Felmeddelande");

/* sätter kortets status till ogiltigt */
ValidationService.validateElement(element);
```

Allt det där finns i exemplet såklart men exemplet är över 80 rader långt och förklarar egentligen ingenting, om man inte vet vad som krävs så är det en hel del detektivarbete för att veta att ta reda på vad som behövs och inte behövs. Låg signal-to-noise ratio.

Reply:
- [ ] Uppdatera dok